mods/d2k/weapons/debris.yaml

```diff
@@ -71,6 +71,35 @@ Debris3:
 	Warhead@4Concrete: DamagesConcrete
 		Damage: 1350
 
+NukeDebris:
```

Comment:
Why `NukeDebris`, and not `DeathHandCluster` or similar? The original game used `DHBOMB` and `EX_DHM` for this (#7558).